form.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+    <link rel="StyleSheet" href="style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+    <title>Конкурс PINK</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <nav class="page-header__nav nav">
+        <div class="nav__menu--prewiew">
+          <a class="nav__logo">
+            <img class="nav__logo-image" src="" alt="Турагенство PINK" width="" height="">
+          </a>
+          <button class="nav__toggle" type="button">Меню</button>
+        </div>
+          <ul class="nav__list">
+          <li class="nav__list--item"><a href="index.html" class="nav__list__link">Главная</a></li>
+          <li class="nav__list--item"><a href="photo.html" class="nav__list__link">Фотографии</a></li>
+          <li class="nav__list--item"><a href="form.html" class="nav__list__link">Конкурс</a></li>
+          <li class="nav__list--item"><a href="#" class="nav__list__link">HTML Академия</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1 class="visually-hidden">Турагенство PINK</h1>
+      <section class="banner">
+        <h2 class="banner-header">Конкурс на самое безбашенное путешествие с Пинк</h2>
+      </section>
+    </main>
+    <footer class="footer">
+      <a class="footer__logo">
+        <img class="footer__logo-img" src="" alt="Турагенство PINK" width="" height="">
+      </a>
+      <div class="footer__social social">
+        <ul class="social-list">
+          <li class="social-item"><a class="social-link social-link--twitter" href="#">Twitter</a></li>
+          <li class="social-item"><a class="social-link social-link--facebook" href="#">Facebook</a></li>
+          <li class="social-item"><a class="social-link social-link--youtube" href="#">Youtube</a></li>
+        </ul>
+      </div>
+      <p class="footer-made-in">Разработано <a class="footer__made-in--link" href="https://htmlacademy.ru/intensive/adaptive">HTML Academy</a></p>
```

Comment:
```suggestion
      <p class="footer__made-in">Разработано <a class="footer__made-in-link" href="https://htmlacademy.ru/intensive/adaptive">HTML Academy</a></p>
```

Три тире, это модификатор. В этом случае модификатор не нужен.